ezbroWorld/missing-number.py

```diff
@@ -0,0 +1,8 @@
+class Solution:
+    def missingNumber(self, nums: List[int]) -> int:
+        tmpArr=nums[:]
+        tmpArr.sort()
+        for i in range(len(tmpArr)):
```

Comment:
```suggestion
        for i in range(len(tmpArr)+1):
```
이케 하면 8줄 return 안 해도 될듓????? 구웃

Reply:
out of range 여따.. 제송

Reply:
out of range 에러가 발생하지 않을까 싶습니다.